src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,777 @@
+            "range.min": "minimum",
+            "range.max": "maximum",
+            "range.step": "divisibleBy"
+        }
+    },
+
+    range: {
+        min: 1,
+        max: 10,
+        step: 1
+    },
+
+    selectors: {
+        radioVolume: ".awec-radioVolume"
+    },
+
+    // more will go here
+
+});
+```
+
+Finally, we need to define the Renderer protoTree – the instructions for rendering
+the model value into the template. This protoTree will need to be a little bit more complicated
+than what we used for the heated seats preference. That protoTree needed to do only one thing:
+bind the model value to the input element. For the range input, we still need to bind the model
+value, but we _also_ need to set the `min`, `max` and `step` attributes of the element. For this,
+a simple key/value pair isn’t enough. Instead of a simple string reference as the value,
+we’ll use an object with a value property:
+
+```javascript
+    protoTree: {
+        radioVolume: {
+          value: "${radioVolume}"
+        }
+    }
+});
+```
+
+
+To this, we will add a _[Renderer Decorator](../RendererDecorators.md)_
+to set the attributes using the contents of the `range` property:
+<aside class="infusion-docs-callout">
+[Renderer Decorators](../RendererDecorators.md) allow users of the Renderer to attach various things,
+such as functions, class names, etc., to the components at render time.
+</aside>
+
+```javascript
+    protoTree: {
+        radioVolume: {
+          value: "${radioVolume}",
+          decorators: [{
+              type: "attrs",
+              attributes: {
+                  min: "{that}.options.range.min",
+                  max: "{that}.options.range.max",
+                  step: "{that}.options.range.step"
+              }
+          }]
+        }
+    }
+});
+```
+
+So, this is what our final radio volume Panel component definition looks like:
+
+```javascript
+fluid.defaults("awesomeCars.prefs.panels.radioVolume", {
+    gradeNames: ["fluid.prefs.panel"],
+
+    preferenceMap: {
+        "awesomeCars.prefs.radioVolume": {
+            "model.radioVolume": "default",
+            "range.min": "minimum",
+            "range.max": "maximum",
+            "range.step": "divisibleBy"
+        }
+    },
+
+    range: {
+        min: 1,
+        max: 10,
+        step: 1
+    },
+
+    selectors: {
+        radioVolume: ".awec-radioVolume"
+    },
+
+    protoTree: {
+        radioVolume: {
+          value: "${radioVolume}",
+          decorators: [{
+              type: "attrs",
+              attributes: {
+                  min: "{that}.options.range.min",
+                  max: "{that}.options.range.max",
+                  step: "{that}.options.range.step"
+              }
+          }]
+        }
+    }
+});
+```
+
+### Adding the Panel to the Editor ###
+
+We saw above that the main HTML template for the tool, `html/prefsEditorTemplate.html`,
+has a placeholder in it for the heated seats Panel
```

Comment:
I think you are missing a period or some type of punctuation at the end of this line after "Panel"